pkgs/development/python-modules/txdbus/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, buildPythonPackage, fetchPypi, six, twisted }:
+
+buildPythonPackage rec {
+  pname = "txdbus";
+  version = "1.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0z41n1ikpdvk0nm8dbyh6g9bg781q4j6hg2b09b5k4wdqm17zxbg";
+  };
+
+  propagatedBuildInputs = [ six twisted ];
```

Comment:
```suggestion
  propagatedBuildInputs = [ six twisted ];
  pythonImportsCheck = [ "txdbus" ];
```

Reply:
Thanks a lot for your feedback.

Unfortunately the tests in the git repository are failing. So I have implemented your second suggestion. Furthermore, I tested the library successfully - both in `bleak` and a small own testing program.